web/src/routes/admin/users/[id]/+page.svelte

```diff
@@ -355,6 +363,56 @@
             {/if}
           </CardBody>
         </Card>
+        <Card color="secondary">
+          <CardHeader>
+            <div class="flex items-center gap-2 px-4 py-2 text-primary">
+              <Icon icon={mdiAppsBox} size="1.5rem" />
+              <CardTitle>Mobile Devices</CardTitle>
+            </div>
+          </CardHeader>
+          <CardBody>
+            <div class="px-4 pb-7">
+              <Stack gap={3}>
+                {#if user.mobileDevices && user.mobileDevices.length > 0}
+                  {#each user.mobileDevices as device (`${device.deviceType}-${device.deviceOS}-${device.lastSeen}`)}
+                    <div
+                      class="flex items-center gap-4 p-3 rounded-lg bg-gray-50 dark:bg-immich-dark-primary/10 shadow-sm"
+                    >
+                      <span class="flex items-center justify-center">
+                        {#if device.deviceType === 'Android' || device.deviceOS === 'Android'}
+                          <Icon icon={mdiAndroid} size="2rem" class="text-green-600" />
+                        {:else if device.deviceType === 'iOS' || device.deviceType === 'Apple' || device.deviceType === 'iPhone' || device.deviceType === 'iPad' || device.deviceOS === 'iOS' || device.deviceOS === 'Apple' || device.deviceOS === 'iPhone' || device.deviceOS === 'iPad' || device.deviceOS === 'macOS'}
```

Comment:
I would have a list `const deviceTypes = ['iOS', 'Apple', 'iPhone', 'iPad', 'macOS'];` and then `if deviceTypes.includes(device.deviceType);`